test/testdata/sample.html

```diff
@@ -1,56 +1,55 @@
-    <meta name='ocr-recognized' value='lines text'>
-    <meta name='DC.creator' value='James T. Kirk'>
-    <meta name='DC.title' value='Nomad'>
-    <meta name='DC.publisher' value='NASA'>
+  <meta name='ocr-id' value='OCRopus Revision: 312'>
+  <meta name='ocr-recognized' value='lines text'>
+  <meta name='DC.creator' value='James T. Kirk'>
+  <meta name='DC.title' value='Nomad'>
+  <meta name='DC.publisher' value='NASA'>
 </head>
 <body>
-<div class='ocr_page' title='file alice_1.png'>
-    <h3><span class='ocr_line' title='bbox 467 525 1386 588'>1 Down the Rabbit-Hole</span>
-</h3>
-<span class='ocr_line' title='bbox 461 648 2077 707'>Alice was beginning to get very tired of sitting by her sister on the bank,</span>
-<span class='ocr_line' title='bbox 461 708 2079 766'>and of having nothing to do: once or twice she had peeped into the book her</span>
-<span class='ocr_line' title='bbox 460 770 2078 826'>sister was reading, but it had no pictures or conversations in it, `and what is</span>
-<span class='ocr_line' title='bbox 459 829 1906 884'>the use of a book,' thought Alice `without pictures or conversation?</span>
+  <div class='ocr_page' title='file alice_1.png'>
+    <h3><span class='ocr_line' title='bbox 467 525 1386 588'>1 Down the Rabbit-Hole</span></h3>
+    <span class='ocr_line' title='bbox 461 648 2077 707'>Alice was beginning to get very tired of sitting by her sister on the bank,</span>
+    <span class='ocr_line' title='bbox 461 708 2079 766'>and of having nothing to do: once or twice she had peeped into the book her</span>
+    <span class='ocr_line' title='bbox 460 770 2078 826'>sister was reading, but it had no pictures or conversations in it, `and what is</span>
+    <span class='ocr_line' title='bbox 459 829 1906 884'>the use of a book,' thought Alice `without pictures or conversation?</span>
     <p class='ocr_par'>
-<span class='ocr_line' title='bbox 533 888 2077 946'>So she was considering in her own mind (as well as she could, for the hot</span>
-<span class='ocr_line' title='bbox 459 947 2079 1007'>day made her feel very sleepy and stupid), whether the pleasure of making a</span>
-<span class='ocr_line' title='bbox 459 1008 2075 1067'>daisy-chain would be worth the trouble of getting up and picking the daisies,</span>
-<span class='ocr_line' title='bbox 458 1069 1819 1127'>when suddenly a White Rabbit with pink eyes ran close by her.</span>
+      <span class='ocr_line' title='bbox 533 888 2077 946'>So she was considering in her own mind (as well as she could, for the hot</span>
+      <span class='ocr_line' title='bbox 459 947 2079 1007'>day made her feel very sleepy and stupid), whether the pleasure of making a</span>
+      <span class='ocr_line' title='bbox 459 1008 2075 1067'>daisy-chain would be worth the trouble of getting up and picking the daisies,</span>
+      <span class='ocr_line' title='bbox 458 1069 1819 1127'>when suddenly a White Rabbit with pink eyes ran close by her.</span>
     <p class='ocr_par'>
```

Comment:
Actually, I would expect that the `p` tags should be closed again. Does the specs say anything about this?